src/components/CheckboxGroup/readme.md

```diff
@@ -28,6 +28,7 @@ class CheckboxGroupTry extends React.Component {
                 <CheckboxGroup
                     id="checkbox-group-1"
                     label="Checkbox Group Label"
+                    labelAlignment="left"
```

Comment:
^^ then you don't need to do this, also note that since this is a new prop if we set it to 'center' as default then it will cause a visual breaking change